spec/03-plugins/20-hmac-auth/03-access_spec.lua

```diff
@@ -108,6 +108,21 @@ describe("Plugin: hmac-auth (access)", function()
       assert.equal(SIGNATURE_NOT_VALID, body.message)
     end)
 
+    it("should not be authorized when the HMAC signature is not properly base64 encoded", function()
+      local date = os.date("!%a, %d %b %Y %H:%M:%S GMT")
+      local res  = assert(client:send {
+        method          = "POST",
+        headers         = {
+          ["HOST"]      = "hmacauth.com",
+          date          = date,
+          authorization = "QypC1WA4v3Q S2chf5Qb+cCheLWk="
+        }
+      })
+      local body = assert.res_status(403, res)
+      body = cjson.decode(body)
+      assert.equal(SIGNATURE_NOT_VALID, body.message)
+    end)
```

Comment:
noting here for posterity because github doesnt track patchset versions in PR with amended commits:

@thibaultcha's comment refers to the face that previously the `authorization` header was not a properly encoded base64 value. the purpose of the test is to determine the behavior of the module when the `signature` portion of the header is not properly encoded, not that the whole header value is an invalid encoding.